lib/3scale/client.rb

```diff
@@ -122,20 +124,27 @@ def authrep(options)
     #
     # == Examples
     #
-    #   # Report two transactions of two applications.
-    #   client.report({:app_id => 'foo', :usage => {'hits' => 1}},
-    #                 {:app_id => 'bar', :usage => {'hits' => 1}})
+    #    Report two transactions of two applications. Using the default service.
+    #    client.report(transactions: [{:app_id => 'foo', :usage => {'hits' => 1}},
+    #                                 {:app_id => 'bar', :usage => {'hits' => 1}}])
     #
-    #   # Report one transaction with timestamp.
-    #   client.report({:app_id    => 'foo',
-    #                  :timestamp => Time.local(2010, 4, 27, 15, 14),
-    #                  :usage     => {'hits' => 1})
+    #    Report one transaction with timestamp. Using the default service.
+    #    client.report(transactions: [{:app_id    => 'foo',
+    #                                  :timestamp => Time.local(2010, 4, 27, 15, 14),
+    #                                  :usage     => {'hits' => 1}])
     #
-    def report(*transactions)
-      raise ArgumentError, 'no transactions to report' if transactions.empty?
+    #    Report a transaction specifying the service.
+    #    client.report(transactions: [{:app_id => 'foo', :usage => {'hits' => 1}}],
+    #                  service_id: 'a_service_id')
+    #
+    def report(transactions: [], service_id: nil)
```

Comment:
That would make the code uglier and the interface less intuitive, but we would not break the compatibility with previous versions. I see that we are not encouraging locking to a major version in the readme: `gem '3scale_client'` so it might be a problem for some users...

I'm thinking that it might be better to do this in a backwards compatible way. What do you think @mayorova ? @mpguerra ?